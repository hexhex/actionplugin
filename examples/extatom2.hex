prefix("test:").
more("a","b","c").
complete(F) :- more(X,Y,Z),prefix(S),
                  &testConcat[S," "](A),
                  &testConcat[A,X](B),
                  &testConcat[B," "](C),
                  &testConcat[C,Y](D),
                  &testConcat[D," "](E),
                  &testConcat[E,Z](F).
