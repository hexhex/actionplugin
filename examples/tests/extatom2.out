{complete("test: a b c"), more("a","b","c"), prefix("test:")}
